query/Makefile

```diff
@@ -1,61 +0,0 @@
-include ../config.mk
```

Comment:
We still need the makefile, it has targets for uploading a jar and running ipython or pytests with the right env vars. We might want a different solution long-term, but for now this is how that happens.